http.go

```diff
@@ -0,0 +1,119 @@
+package goutils
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"github.com/Yapo/goutils/commonErrors"
+	"github.com/Yapo/logger"
+	"io/ioutil"
+	"net/http"
+	"strconv"
+	"strings"
+)
+
+// Response is a struct to generate a response from POST/PUT requests
+type Response struct {
+	Code int
+	Body interface{}
+}
+
+// ErrorType allow have common error on diferent projects
+type ErrorType struct {
+	Code    int
+	Message string
+}
+
+// WriteHTTPResponse write to te response stream
+func WriteHTTPResponse(w http.ResponseWriter, response *Response) {
```

Comment:
I would call this one `WriteJSONResponse`, as the content type us fixed to `application/json`